tests/trondaemon_test.py

```diff
@@ -78,3 +78,30 @@ def setup(self):
     @teardown
     def teardown(self):
         self.tmpdir.cleanup()
+
+    def test_make_sigint_handler_keyboardinterrupt(self):
+        daemon = TronDaemon.__new__(TronDaemon)  # skip __init__
```

Comment:
You can use @mock.patch.object(TronDaemon, _handle_shutdown) to mock out the method instead. Then you should be able to use self.trond in the test.

(same for the test below)

Reply:
But part of creating `self.trond` is that it uses `_make_sigint_handler` during initialization. Doesn't it go against the spirit of testing to use an object to test something we have already used without testing?